src/NzbDrone.Core/Music/Track.cs

```diff
@@ -24,14 +24,17 @@ public Track()
         public int ArtistId { get; set; } // This is the DB Id of the Artist, not the SpotifyId
         //public int CompilationId { get; set; }
         public bool Compilation { get; set; }
-        public int TrackNumber { get; set; }
+        public string TrackNumber { get; set; }
+        public int AbsoluteTrackNumber { get; set; }
+        public int RelativeTrackNumber { get; set; }
```

Comment:
Shouldn't relative number be alphanumeric or are we doing absolute position based on medium?

Reply:
Removed relative